app/src/main/java/com/pnow/rick_and_morty_list/app/data/mapper/DetailsUiMapper.kt

```diff
@@ -27,10 +28,12 @@ class DetailsUiMapper @Inject constructor(val context: Context) {
     }
 
     fun mapToEpisodeUIModel(episodeDto: EpisodeModel): EpisodeUIModel {
```

Comment:
fun mapToEpisodeUIModel(episodeDto: EpisodeModel) =EpisodeUIModel(
            id = episodeDto.id,
            episodeNumber = episodeDto.episode,
            name = episodeDto.name,
            date = episodeDto.airDate,
        )


mniej kodu

Reply:
ale dobra widzę, że masz po prostu taką konwecję to zostaw